src/CookieBanner/CookieBanner.vue

```diff
@@ -4,22 +4,24 @@
       <div class="banner-wrapper" v-if="show">
         <div class="wrapper">
           <div class="message">By using this website, you agree to our <a href="https://www.cookiesandyou.com" target="_blank" rel="noopener" style="color: #505FFF;">Cookie Policy</a>.</div>
-          <span class="icon-cross" @click="close">
-            <svg
-              width="14"
-              height="14"
-              viewBox="0 0 14 14"
-              fill="none"
-              xmlns="http://www.w3.org/2000/svg"
-            >
-              <path
-                d="M1.66669 1.66669L12.3334 12.3334M12.3334 1.66669L1.66664 12.3334"
-                stroke="#A2A3AD"
-                stroke-width="1.5"
-                stroke-linecap="round"
-              />
-            </svg>
-          </span>
+            <div class="box">
```

Comment:
The `@click="close"` and any other hover/active/focus states styling should be tied to this bigger container element, since that's the size of target area we need.